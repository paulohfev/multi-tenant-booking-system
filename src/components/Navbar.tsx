import Image from "next/image";
import Link from "next/link";

const Navbar: React.FC = () => {
  return (
    <nav className="bg-white shadow-md">
      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
        <div className="flex justify-between items-center h-16">
          <div className="flex items-center">
            <Link href="/centers" className="flex items-center text-gray-900 hover:text-gray-700 px-3 py-2 rounded-md text-sm font-medium">
              <div className="flex-shrink-0">
                <Image src="/logos/company.svg" alt="Beauty Center" width={32} height={32} />
              </div>
              <div className="flex items-baseline space-x-4 ml-2">
                Multi-tenant Booking System
              </div>
            </Link>
          </div>
        </div>
      </div>
    </nav>
  )
}

export default Navbar;