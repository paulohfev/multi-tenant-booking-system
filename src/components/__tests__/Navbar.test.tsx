import { render } from '@testing-library/react'
import Navbar from '../Navbar'

describe('Navbar', () => {
  it('should match snapshot', () => {
    const { container } = render(<Navbar />)
    expect(container).toMatchSnapshot()
  })
})
