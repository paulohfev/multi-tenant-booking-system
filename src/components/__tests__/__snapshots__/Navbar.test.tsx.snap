// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<div>
  <nav
    class="bg-white shadow-md fixed w-full top-0 z-50"
  >
    <div
      class="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8"
    >
      <div
        class="flex justify-between items-center h-16"
      >
        <div
          class="flex items-center"
        >
          <a
            class="flex items-center text-gray-900 hover:text-gray-700 px-3 py-2 rounded-md text-sm font-medium"
            href="/centers"
          >
            <div
              class="flex-shrink-0"
            >
              <img
                alt="Beauty Center"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/logos/company.svg"
                style="color: transparent;"
                width="32"
              />
            </div>
            <div
              class="flex items-baseline space-x-4 ml-2"
            >
              Multi-tenant Booking System
            </div>
          </a>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
